exhibitor/migrations/0003_user_is_active.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.1.1 on 2023-02-08 20:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
this change is not a part from LEGO-5 but LEGO-2. The reason why we see it is that you probably created branch from LEGO-2.

what you can do now:
go to main branch
run:
checkout -b LEGO-5_new
git cherry-pick b326601b170327f2a8eeab88fb4a05d784cb31f5

cherry pick will copy commit from LEGO-5 branch to LEGO-5_new branch (this proper commit)

see your commit
https://github.com/Bialy12344/LEGO_Exhibition/pull/10/commits/b326601b170327f2a8eeab88fb4a05d784cb31f5

more about cherry-pick https://git-scm.com/docs/git-cherry-pick